drivers/ethernet/Kconfig.dsa

```diff
@@ -6,7 +6,7 @@
 
 menuconfig NET_DSA
 	bool "Distributed Switch Architecture support"
-	depends on ETH_MCUX || ETH_SAM_GMAC
+	depends on ETH_MCUX || ETH_SAM_GMAC || ETH_STM32_HAL
 	help
 	  Enable Distributed Switch Architecture support. For now it
 	  only supports Kinetics ENET driver.
```

Comment:
Could you adjust the help text as we have now more supported drivers.

Reply:
Sure